site-templates/fragments/navigation.gohtml

```diff
@@ -18,18 +19,49 @@
                 {{$class = "c-nav__selected"}}
             {{end}}
             {{if $element.Metadata.HasDoc}}
-            <li>
-                <a class="{{$class}}" href="{{$.Prefix}}doc/{{$svc.Namespace}}/{{$svc.Name}}">
-                    {{$svc.Namespace}}.{{$svc.Name}}
-                </a>
-            </li>
+                {{if or (eq $svc.Name "elementary") (eq $svc.Name "isam")}}
+                    <li>
+                        <a class="{{$class}}" href="{{$.Prefix}}doc/{{$svc.Namespace}}/{{$svc.Name}}">
+                            {{$svc.Namespace}}.{{$svc.Name}}
+                        </a>
+                    </li>
+                {{else}}
+                    <li><samp>None</samp></li> 
+                {{end}}
+            {{end}}
+        {{end}}
+        {{if not $haveApi}}
+        <li><samp>None</samp></li>
+        {{end}}
+        </ul>
+    </div>
+    <div>
+        <strong>Bluescape v2 APIs</strong>
+        <ul class="unindent-side-nav-ul">
+        {{range $i, $element := .S.K8sStore.Slice }}
+            {{$svc := $element.Service}}
+            {{$class := ""}}
+            {{if and (eq $.CurrentService $svc.Name) (eq $.CurrentNamespace $svc.Namespace)}}
+                {{$class = "c-nav__selected"}}
             {{end}}
+            {{if $element.Metadata.HasDoc}}
+                {{if or (eq $svc.Name "collab") (eq $svc.Name "identity-api") (eq $svc.Name "portal-api") (eq $svc.Name "listener-api")}}
+                <li>
+                    <a class="{{$class}}" href="{{$.Prefix}}doc/{{$svc.Namespace}}/{{$svc.Name}}">
+                        {{$svc.Namespace}}.{{$svc.Name}}
```

Comment:
In theory, we could printout whatever we want here. Good, to keep in mind to add customized names for the documents.

Reply:
Good point. Yes, we can easily put whatever name we want instead of "staging.collab" for example